# urr090
URR 0.9.0

## World 1D

~~~~
Traceback (most recent call last):
  File "URR090.py", line 950235, in <module>
  File "URR090.py", line 937867, in place_menu
  File "URR090.py", line 103912, in world_browser
  File "URR090.py", line 931769, in new_load_game
  File "URR090.py", line 935562, in new_play_game
  File "URR090.py", line 901217, in keypresses
  File "URR090.py", line 707873, in player_action
  File "URR090.py", line 135387, in next_turn
  File "URR090.py", line 175729, in do_turn
  File "URR090.py", line 158757, in do_something
  File "URR090.py", line 710015, in find_a_schedule_target
  File "random.py", line 261, in choice
IndexError: Cannot choose from an empty sequence
[7108] Failed to execute script URR090
~~~~

### 0.9.0b

~~~~
Traceback (most recent call last):
  File "URR090b.py", line 950243, in <module>
  File "URR090b.py", line 937869, in place_menu
  File "URR090b.py", line 104022, in world_browser
  File "URR090b.py", line 111265, in character_creation_1
  File "URR090b.py", line 933472, in new_new_game
  File "URR090b.py", line 935570, in new_play_game
  File "URR090b.py", line 901225, in keypresses
  File "URR090b.py", line 707881, in player_action
  File "URR090b.py", line 135360, in next_turn
  File "URR090b.py", line 175737, in do_turn
  File "URR090b.py", line 164807, in do_something
UnboundLocalError: local variable 'oldroad' referenced before assignment
~~~~
